### PR TITLE
adding rails kubernetes template

### PR DESCRIPTION
**Pull request by coryodaniel**
This is good to go; it will stand up a rails app, perform db creation/migration during a job.

### CODE REVIEW

**Comment by coryodaniel** [comment]
We are going to move secrets and SAs to regular resources not hooks. Instead we will use genreateName on jobs to make unique jobs for each apply.

this will delete the secrets and SA on decomm instead ofleaving these sensitive resource around...

the downside is a _lot_ of jobs will be left around... if we use `ttlafterfinished` https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/ k8s will clean it up...

**Review by chrisghill — APPROVED** [review]
LGTM

File: rails-kubernetes/src/main.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module "helm" {
+  source             = "github.com/massdriver-cloud/terraform-modules//massdriver-application-helm?ref=a1a0d8a563e7c0b2a4cdde7654b047a77a1effc5"
+  name               = var.md_metadata.name_prefix
+  namespace          = var.namespace
+  chart              = "ruby-on-rails"
+  helm_repository    = "https://massdriver-cloud.github.io/helm-charts/"
+  helm_version       = "0.1.0"
```
Comment (coryodaniel) [inline rails-kubernetes/src/main.tf]:
```suggestion
  helm_version       = "0.2.0"
```

File: rails-kubernetes/src/main.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module "helm" {
+  source             = "github.com/massdriver-cloud/terraform-modules//massdriver-application-helm?ref=a1a0d8a563e7c0b2a4cdde7654b047a77a1effc5"
+  name               = var.md_metadata.name_prefix
+  namespace          = var.namespace
+  chart              = "ruby-on-rails"
+  helm_repository    = "https://massdriver-cloud.github.io/helm-charts/"
+  helm_version       = "0.2.0"
```
Comment (coryodaniel) [inline rails-kubernetes/src/main.tf]:
```suggestion
  helm_version       = "0.2.1"
```

**Pull request merged**